src/sdk/namespace_cache.js

```diff
@@ -259,7 +264,9 @@ class NamespaceCache {
                             xattr: object_info_hub.xattr,
                             last_modified_time: (new Date(object_info_hub.create_time)).getTime(),
                         };
-                        const upload_res = await this.namespace_nb.upload_object(upload_params, object_sdk);
+
+                    var start = process.hrtime();    
+                    const upload_res = await this.namespace_nb.upload_object(upload_params, object_sdk).then(() => this._update_metric('set_hub_read_latency', (process.hrtime()[1] - start[1])/1000000));
```

Comment:
A couple of issues:
1. this code lives in an async function so there are no reasons the use `.then()`, using `then()` will also change the return value that is assigned to the `upload_res` variable
2. `process.hrtime` can take a param of prev invocation of `process.hrtime` and will provide the delta itself.

The code can be written as follows:
```
const start = process.hrtime();    
const upload_res = await this.namespace_nb.upload_object(upload_params, object_sdk);
this._update_metric('set_hub_read_latency', process.hrtime(start)/1000000));
```

Please note that this comment should also apply to all changes made to the file below